server/api/utils/email.ts

```diff
@@ -0,0 +1,63 @@
+import nodemailer from "nodemailer";
+
+interface Item {
+    name: string;
+    quantity: number;
+}
+async function sendApprovalEmail(firstName: string, lastName: string, email: string, item: Item) {
+    const transporter = nodemailer.createTransport({
+        service: "gmail",
+        auth: {
+            user: "blueprintpregnancycentre@gmail.com",
+            pass: "pregnancycentre"
+        }
+    });
+    let htmlString = `<body><p>Dear ${firstName} ${lastName}, <p>`;
+    htmlString += `<p>Your donation form of ${item.quantity} ${
+        item.name + (item.quantity == 1 ? "" : "s")
+    } has been approved. Contact rebecca@pregnancycentre.ca to make further emails.</p>`;
+
+    htmlString += "The Pregnancy Centre</body>";
+
+    const info = await transporter.sendMail({
+        from: '"no reply " <no-reply@pregnancycentre.ca>', // sender address
+        to: email, // list of receivers
+        subject: "Pregnancy Centre: Donation form approval", // Subject line
+        text: "Thank you for submitting a donation form.", // plain text body
+        html: htmlString // html body
+    });
+
+    console.log("Message sent: %s", info.messageId);
+    // console.log("Preview URL: %s", nodemailer.getTestMessageUrl(info));
+}
+
+async function sendConfirmationEmail(firstName: string, lastName: string, email: string, items: Array<Item>) {
+    const transporter = nodemailer.createTransport({
+        service: "gmail",
+        auth: {
+            user: "blueprintpregnancycentre@gmail.com",
+            pass: "pregnancycentre"
+        }
+    });
```

Comment:
would it be better to initialize this outside of the function? it seems duplicated between the two send email functions

Reply:
Updated.